ui/src/routes/lecture.tsx

```diff
@@ -67,13 +67,21 @@ function Lecture() {
               <Title level={3}>{lecture.speaker}</Title>
               <Title>{lecture.title}</Title>
               <div className="details">
-                <Title level={4}>{lecture.date}</Title>
-                <div id="bullet">•</div>
-                <a href={lecture.event_details}>Event details (Indico)</a>
-                <div id="bullet">•</div>
-                <Title level={4}>
-                  Sponsored by <strong>{lecture.sponsor}</strong>
-                </Title>
+                {lecture.date && lecture.date.length > 0 && (
+                  <Title level={4} id="date">
+                    {lecture.date}
+                  </Title>
+                )}
+                {lecture.event_details && lecture.event_details.length > 0 && (
+                  <a href={lecture.event_details} id="event">
+                    Event details (Indico)
+                  </a>
+                )}
+                {lecture.sponsor && lecture.sponsor.length > 0 && (
+                  <Title level={4} id="sponsor">
+                    Sponsored by <strong>{lecture.sponsor}</strong>
+                  </Title>
+                )}
```

Comment:
```suggestion
                {lecture.date && lecture.date !== '' > 0 && (
                  <Title level={4} id="date">
                    {lecture.date}
                  </Title>
                )}
                {lecture.event_details && lecture.event_details !== '' > 0 && (
                  <a href={lecture.event_details} id="event">
                    Event details (Indico)
                  </a>
                )}
                {lecture.sponsor && lecture.sponsor !== '' > 0 && (
                  <Title level={4} id="sponsor">
                    Sponsored by <strong>{lecture.sponsor}</strong>
                  </Title>
                )}
```

:)

Reply:
@drjova you mean without `> 0`?

Reply:
> :)

because this way it won't work

Reply:
`{lecture.date && lecture.date !== '' && (
                  <Title level={4} id="date">
                    {lecture.date}
                  </Title>
                )}
                {lecture.event_details && lecture.event_details !== '' && (
                  <a href={lecture.event_details} id="event">
                    Event details (Indico)
                  </a>
                )}
                {lecture.sponsor && lecture.sponsor !== '' && (
                  <Title level={4} id="sponsor">
                    Sponsored by <strong>{lecture.sponsor}</strong>
                  </Title>
                )}
`
let me know if that's what you meant

Reply:
without the `>0` sorry

Reply:
Okay, I make a new commit then.